client/ayon_ftrack/plugins/publish/collect_webpublisher_credentials.py

```diff
@@ -53,11 +53,6 @@ def process(self, context):
         username = self._get_ftrack_username(user_email)
         os.environ["FTRACK_API_USER"] = username
 
-        burnin_name = username
```

Comment:
I don't think we want to remove this. I think this solves cases where customer's artists have their email address as username (john.doe@example.com), they don't want email part in burnin.

Reply:
That should not be a job of ftrack addon in AYON. In AYON should be `context.data["user"]` always filled with AYON username. Created issue in webpublisher https://github.com/ynput/ayon-webpublisher/issues/13 .